lms/static/js/ccx/schedule.js

```diff
@@ -104,23 +104,37 @@ var edx = edx || {};
 	    // Add unit handler
 	    $('#add-unit-button').on('click', function(event) {
 	      event.preventDefault();
+	      // Default value of time is 00:00.
+	      var start = self.get_datetime('start');
```

Comment:
@justinabrahms I think this is ok to go check edX standards for javascript https://github.com/edx/edx-platform/wiki/Javascript-standards-for-the-edx-platform